docs/advanced_guide/cloud/azure/notebooks/multi_node_search/scripts/commands.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import json
+from azure.ai.ml import command
+from azure.ai.ml import Input, Output
+from azure.ai.ml.entities import UserIdentityConfiguration
+
+
+def make_train_model_command(output_path, code_dir, environment_name, id,
```

Comment:
Should there be readme file under multi_node_search/scripts to explain how these scripts are organized?

Reply:
Good idea, I'll add it to the main readme though, is that good enough, or do we have a convention of one readme per folder?

Reply:
Done.